src/SkeletonGenerator.ump

```diff
@@ -9,39 +13,45 @@ class SkeletonGenerator {
     
     internal Path pathRelativeTo;
     internal String fileFrom;
+    lazy boolean hasSpecializedSkeleton;
```

Comment:
I believe so, since the constructor shouldn't accept it as an argument. Is instead determined based on the fileFrom property.